doc/man7/EVP_KDF-HKDF.pod

```diff
@@ -15,67 +15,41 @@ and "extracts" from it a fixed-length pseudorandom key K. The second stage
 "expands" the key K into several additional pseudorandom keys (the output
 of the KDF).
 
-=head2 Numeric identity
+=head2 Identity
 
-B<EVP_KDF_HKDF> is the numeric identity for this implementation; it
-can be used with the EVP_KDF_CTX_new_id() function.
+B<SN_HKDF> is the name for this implementation; it
+can be used with the EVP_KDF_new() function.
 
-=head2 Supported controls
+=head2 Supported parameters
 
-The supported controls are:
+The supported parameters are:
 
 =over 4
 
-=item B<EVP_KDF_CTRL_SET_SALT>
+=item B<OSSL_KDF_PARAM_PROPERTIES> ("properties") <UTF8 string>
 
-=item B<EVP_KDF_CTRL_SET_MD>
+=item B<OSSL_KDF_PARAM_DIGEST> ("digest") <UTF8 string>
 
-=item B<EVP_KDF_CTRL_SET_KEY>
+=item B<OSSL_KDF_PARAM_KEY> ("key") <octet string>
 
-These controls work as described in L<EVP_KDF_CTX(3)/CONTROLS>.
+=item B<OSSL_KDF_PARAM_SALT> ("salt") <octet string>
 
-=item B<EVP_KDF_CTRL_RESET_HKDF_INFO>
+These parameters work as described in L<EVP_KDF_CTX(3)/PARAMETERS>.
 
-This control does not expect any arguments.
+=item B<OSSL_KDF_PARAM_INFO> ("info") <octet string>
 
-Resets the context info buffer to zero length.
-
-=item B<EVP_KDF_CTRL_ADD_HKDF_INFO>
-
-This control expects two arguments: C<unsigned char *info>, C<size_t infolen>
-
-Sets the info value to the first B<infolen> bytes of the buffer B<info>.  If a
-value is already set, the contents of the buffer are appended to the existing
-value.
-
-The total length of the context info buffer cannot exceed 1024 bytes;
+This parameter sets the info value.
+The length of the context info buffer cannot exceed 1024 bytes;
 this should be more than enough for any normal use of HKDF.
 
-EVP_KDF_ctrl_str() takes two type strings for this control:
-
-=over 4
+=item B<OSSL_KDF_PARAM_MODE> ("mode") <UTF8 string> or <int>
 
-=item "info"
-
-The value string is used as is.
-
-=item "hexinfo"
-
-The value string is expected to be a hexadecimal number, which will be
-decoded before being passed on as the control value.
-
-=back
-
-=item B<EVP_KDF_CTRL_SET_HKDF_MODE>
-
-This control expects one argument: C<int mode>
-
-Sets the mode for the HKDF operation. There are three modes that are currently
-defined:
+This parameter sets the mode for the HKDF operation.
+There are three modes that are currently defined:
 
 =over 4
 
-=item EVP_KDF_HKDF_MODE_EXTRACT_AND_EXPAND
+=item "EXTRACT_AND_EXPAND" (EVP_KDF_HKDF_MODE_EXTRACT_AND_EXPAND)
```

Comment:
The intention here was that the string is the preferred version.  This is the reverse of the parameter names isn't it?

Reply:
I've changed these over since I was bolding things anyway.

I like this format better too.  I think a separate PR to globally make this change would be to way to go.

Reply:
If someone makes a PR to remove all emphasis other than POD's ```B<>``` construct, please update find-doc-nits to complain.

Reply:
> If someone makes a PR to remove all emphasis other than POD's B<> construct

That's not what we're talking about here